lgc/state/PipelineState.cpp

```diff
@@ -1121,7 +1121,63 @@ unsigned PipelineState::getShaderWaveSize(ShaderStage stage) {
   assert(stage <= ShaderStageCompute);
   if (!m_waveSize[stage])
     setShaderDefaultWaveSize(stage);
-  return m_waveSize[stage];
+
+  unsigned waveSize = m_waveSize[stage];
```

Comment:
I think you can write like this

if ( >= 9)
  return getMergedShaderWaveSize(stage);

return m_waveSize[stage];